### ISSUE TITLE
France (National Assembly): protocol of nosdeputes is flip-flopping

### ISSUE THREAD

**Issue by davewhiteland** [everypolitician/everypolitician-data#13199]
It looks like the scraper is sometimes getting `https` returned in the `url_nosdeputes_api` field of the JSON returned, and sometimes `http`, generating 600+ unnecessary changes to the data each time it flips, as a URL in the `sources` in the Popolo file we're generating.

It looks to me like this is coming from the source itself (i.e., `https://www.nosdeputes.fr/deputes/json` itself has the complete URL as a field) so maybe this is something the scraper should explicitly notice and ignore; e.g. picking one protocol and sticking with it), because the scraper is always using the same `https` URL as its entry point.

**Comment by RouxRC** [comment]
Good catch, seems like a cache discrepancy which is shared between http and https calls.
A bit complex to handle properly with current architecture, so until a better fix, we froze all urls returned by the API to be only http now whatever you call.
should be fixed as well on nossenateurs.fr
You don't use the previous legislature's data from 2007.nosdeputes.fr do you?

**Comment by davewhiteland** [comment]
@RouxRC merci for this!  :-)   

And you're right, we're not _currently_ using 2007.nosdeputes.fr as a source for the historic data (that's coming from http://www.assemblee-nationale.fr/sycomore/ ...)
